### PR TITLE
Don't install husky anymore, update docs for husky v8

### PR DESCRIPTION
**Pull request by askoufis**
Sku's init script installs and configures `husky` for you, but it installs the latest husky version (v8), which configures husky in a different way than the configuration provided by sku (old v4 config). So out of the box you get a broken husky experience.

If we wanted to support husky v8 out of the box, we'd have to create a husky folder and script, but I'd prefer users manage that kind of stuff, rather than sku. So I've removed the husky code from sku, and documented the husky v8 setup steps in the docs instead.

### CODE REVIEW

File: .changeset/nine-paws-unite.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+---
+
+`sku init` no longer installs and configures husky for you
+
+BREAKING CHANGE
+
+`sku init` no longer adds `husky` as a dependecy nor does it configure husky for you out of the box.
+For instructions on how to set up `husky` to use sku's pre-commit hook, see [the docs].
+
+[the docs]: https://seek-oss.github.io/sku/#/./docs/extra-features?id=pre-commit-hooks
```
Comment (mrm007) [inline .changeset/nine-paws-unite.md:12]: You changed the heading so this must change as well.

```suggestion
[the docs]: https://seek-oss.github.io/sku/#/./docs/extra-features?id=pre-commit-hook
```

Comment (askoufis) [reply .changeset/nine-paws-unite.md:12]: Good catch 👍

File: docs/docs/extra-features.md [hunk 2]
```diff
@@ -55,23 +58,34 @@ module.exports = {
...
-    "pre-commit": "sku pre-commit"
+{
+  "scripts": {
+    "prepare": "husky install"
   }
-},
+}
+```
+
+```sh
```
Comment (mrm007) [inline docs/docs/extra-features.md:76]: I think we should leave this out and just point to Husky's docs: https://typicode.github.io/husky/#usage. The way Husky manages hooks is an implementation detail and may change in the future.

File: docs/docs/extra-features.md [hunk 3]
```diff
@@ -55,23 +58,34 @@ module.exports = {
...
+# .husky/pre-commit
+
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+yarn sku pre-commit
 ```
```
Comment (mrm007) [inline docs/docs/extra-features.md:83]:
```suggestion
```sh
yarn husky add .husky/pre-commit "yarn sku pre-commit"
```

For more details configuring hooks, please see Husky's [documentation](https://typicode.github.io/husky/#usage).
```

**Review by mrm007 — APPROVED** [review]

File: .changeset/nine-paws-unite.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+---
+'sku': major
+---
+
+`sku init` no longer installs and configures husky for you
+
+BREAKING CHANGE
+
+`sku init` no longer adds `husky` as a dependecy nor does it configure husky for you out of the box.
```
Comment (mrm007) [inline .changeset/nine-paws-unite.md:9]:
```suggestion
`sku init` no longer adds `husky` as a dependecy nor does it configure Husky for you out of the box.
```

old man

File: .changeset/nine-paws-unite.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+For instructions on how to set up `husky` to use sku's pre-commit hook, see [the docs].
```
Comment (mrm007) [inline .changeset/nine-paws-unite.md:10]:
```suggestion
For instructions on how to set up Husky to use sku's pre-commit hook, see [the docs].
```

File: docs/docs/extra-features.md [hunk 6]
```diff
@@ -55,23 +58,31 @@ module.exports = {
 
 `sku` comes with bundle analysis built in via [webpack-bundle-analyzer](https://www.npmjs.com/package/webpack-bundle-analyzer). A report is generated in the `/report` directory when `sku build` is run.
 
-## Pre-commit hooks
+## Pre-commit hook
 
-To speed up the feedback loop on linting and formatting errors, `sku` provides a `pre-commit` script that can be run to catch simple problems before CI. To make use of this, it's recommended that you install [husky](https://www.npmjs.com/package/husky) as a development dependency and configure it in `package.json` as follows:
+To speed up the feedback loop on linting and formatting errors, `sku` provides a `pre-commit` script that can be run to catch simple problems before CI.
+To make use of this hook, it's recommended to install [husky](https://www.npmjs.com/package/husky) as a development dependency and configure it as follows:
```
Comment (mrm007) [inline docs/docs/extra-features.md:64]:
```suggestion
To make use of this hook, it's recommended to install [Husky](https://www.npmjs.com/package/husky) as a development dependency and configure it as follows:
```

File: .changeset/nine-paws-unite.md (hunk 4 above)
Comment (mrm007) [reply .changeset/nine-paws-unite.md:9]: Actually, in this one I'm the old man 😢

File: docs/docs/extra-features.md [hunk 7]
```diff
@@ -55,23 +58,31 @@ module.exports = {
...
   }
-},
+}
+```
+
+```sh
+yarn husky add .husky/pre-commit "yarn sku pre-commit"
 ```
 
+For more details configuring hooks, please see Husky's [documentation](https://typicode.github.io/husky/#create-a-hook).
```
Comment (mrm007) [inline docs/docs/extra-features.md:80]:
```suggestion
For more details configuring hooks, please see husky's [documentation](https://typicode.github.io/husky/#create-a-hook).
```

**Pull request merged**